src/app/showcase/components/message-popup/showcase-message-popup.component.ts

```diff
@@ -34,4 +35,15 @@ export class ShowcaseMessagePopupComponent {
 				console.log('Observable returned to showcase', v);
 			});
 	}
+
+	public showCustomized() {
+		const buttons: MessagePopupButton[] = [];
```

Comment:
You can easily define all the items of the array on initializing stage

Reply:
thanks, done